modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -235,5 +235,188 @@
     "org.wso2.carbon.identity.entitlement.pip.DefaultResourceFinder"
   ],
   "identity.entitlement.entitlement_engine_caching_interval": "100000",
-  "identity.entitlement.JSON_shorten_form_enabled": false
+  "identity.entitlement.JSON_shorten_form_enabled": false,
+
+
+  "identity.events.threadPool_size": "10",
+  "identity.events.schemes": {
+    "account.lock.handler": {
+      "subscriptions": [
+        "PRE_AUTHENTICATION",
+        "POST_AUTHENTICATION",
+        "PRE_SET_USER_CLAIMS",
+        "POST_SET_USER_CLAIMS"
+      ],
+      "properties": {
+        "enable": false,
+        "login.fail.timeout.ratio": "2",
+        "On.Failure.Max.Attempts": "5",
```

Comment:
shouldn't we enclose these in quotes?

Reply:
When we enclose them in single quotes, single quotes also get printed in the jinja template file.

`account.lock.handler.'On.Failure.Max.Attempts'=5`

Reply:
Fixed.